spec/recipes/server_spec.rb

```diff
@@ -85,6 +85,41 @@
     include_examples 'allow weak hmacs'
   end
 
+  context 'with permit_tunnel set to true' do
+    cached(:chef_run) do
+      ChefSpec::ServerRunner.new do |node|
+        node.normal['ssh-hardening']['ssh']['server']['permit_tunnel'] = true
+      end.converge(described_recipe)
+    end
+  end
+
+  context 'with permit_tunnel at deafult value of false' do
+    cached(:chef_run) do
+      ChefSpec::ServerRunner.new do |node|
+        node.normal['ssh-hardening']['ssh']['server']['permit_tunnel'] = false
+      end.converge(described_recipe)
+    end
+  end
+
+  context 'with permit_tunnel set to a valid string' do
+    cached(:chef_run) do
+      ChefSpec::ServerRunner.new do |node|
+        node.normal['ssh-hardening']['ssh']['server']['permit_tunnel'] = 'ethernet'
+      end.converge(described_recipe)
+    end
+  end
+
+  context 'with permit_tunnel set to an invalid string' do
+    cached(:chef_run) do
```

Comment:
probably you have to switch here everywhere to `let(:chef_run)`, otherwise it won't work with changing of attribute